microsoft/surface/common/repos.nix

```diff
@@ -4,8 +4,8 @@
   linux-surface = fetchFromGitHub {
     owner = "linux-surface";
     repo = "linux-surface";
-    rev = "b82e8acd3c015190423b114770b0e9fcc206dd2d";
-    hash = "sha256-parp1F5fFzKkiM6ILK+ZolMdSwU1kLOOMvksSwE/zKA=";
+    rev = "a6eafcad32dc789ae92f42636b11e9aae6e7c879";
```

Comment:
should we also switch to tags here?

Reply:
Yeah, that's probably easier to see which version we're on. I'll likely be doing some more work on this hardware overlay in the near future, I'll change it with the next PR.